test/testunusedvar.cpp

```diff
@@ -4743,6 +4744,47 @@ class TestUnusedVar : public TestFixture {
             "}");
         ASSERT_EQUALS("", errout.str());
     }
+
+    void classWithoutSideEffects()
```

Comment:
We should probably add some very simple negative tests. There's no need to use `main()` as the function. Just using any function without parameter should be fine and simply things a bit.

Reply:
Done.